### PR TITLE
Disable nvdimm_nvml test case on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID: 2018

### CODE REVIEW

**Comment by MiriamDeng** [comment]
@PaulYuuu @maxujun could you help to review this patch ? Thank you !

**Comment by maxujun** [comment]
LGTM，Acked.

File: qemu/tests/cfg/nvdimm.cfg [hunk 1]
```diff
@@ -132,6 +132,7 @@
                     type = boot
                     unarmed_dimm_mem1 = on
         - nvdimm_nvml:
+            no ppc64, ppc64le
```
Comment (PaulYuuu) [inline qemu/tests/cfg/nvdimm.cfg:135]: Do we need to restrict the version to disable?

Comment (MiriamDeng) [reply qemu/tests/cfg/nvdimm.cfg:135]: Per David, we don't need to, thanks.

**Pull request merged**